f4jumble.py

```diff
@@ -123,7 +136,17 @@ def randbytes(l):
             ('normal', 'Vec<u8>'),
             ('jumbled', 'Vec<u8>'),
         ),
-        test_vectors,
+        plain_test_vectors,
+    )
+    print()
+    render_tv(
+        args,
+        'f4jumble',
+        (
+            ('length', 'usize'),
+            ('jumbled_hash', 'Vec<u8>'),
+        ),
+        hashed_test_vectors,
     )
```

Comment:
There is only supposed to be one set of test vectors returned per file. Move this to a new test file `f4jumble_hashed.py`.